W-13731552-reassignorphanedentities-se (#214)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/settings-adminsettings-general.adoc

= Configure General Settings
:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdP). After your organization is migrated to Anypoint Platform, the *Organization Management* module is replaced by the *Settings* module.

> The *General* tab in the *Admin Settings* view enables you to configure the currency and decimal sign to show in RPA Manager modules.

Reviewer: Are you configuring the signs to show up in RPA manager or how the signs show up?

== Before You Begin

* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the required privileges]:
+
[cols="1,2"]
|===
|*Action* |*Required Privileges*

|View configured settings for your organization
|Organization Management Open

|Add or modify settings for your organization
|Organization Management Administration
|===

== Configure Currency and Decimal Sign

To configure the currency and decimal sign to show in RPA Manager modules:

. In the *Settings* module, click *Admin Settings*.

> . Ensure you are in the *General* tab.

Reviewer: ```suggestion
. Ensure that you are in the *General* tab.
```

. Configure the following values:
** *Currency*
+

> The currency symbol RPA Manager shows in modules and dashboards.

Reviewer: ```suggestion
The currency symbol that RPA Manager shows in modules and dashboards.
```

** *Decimal Sign*
+

> The decimal sign RPA Manager uses to show fractional numbers in modules and dashboards.

Reviewer: ```suggestion
The decimal sign that RPA Manager uses to show fractional numbers in modules and dashboards.
```

= Configure General Settings
:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdPs). After your organization migrates to Anypoint Platform, the Settings module replaces the Organization Management module.

The *General* tab in the *Admin Settings* view enables you to configure the currency and decimal sign to show in RPA Manager modules.

== Before You Begin

> * Ask an Anypoint Access Management administrator to assign you the required permissions:

Reviewer 2: ```suggestion
* Ask an organization administrator in Access Management to assign you the required permissions:
```
